checker/tests/nullness/Issue2215.java

```diff
@@ -0,0 +1,34 @@
+// Test Case for Issue2215
+// https://github.com/typetools/checker-framework/issues/2215
+// @skip-test
+
+import org.checkerframework.checker.nullness.qual.MonotonicNonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class ParentClass {
+
+    @MonotonicNonNull Object parentClassField;
+
+    ParentClass(@Nullable Object o) {
+        this.parentClassField = o; // should be allowed
```

Comment:
You don't need "should be allowed" text.  Anything that is not preceded by a `// ::` comment is allowed, and this text clutters the test case.